### ISSUE TITLE
FORCE and FORCEALL do the exact same thing

### ISSUE THREAD

**Issue by demonbane** [mrworf/plexupdate#170]
I didn't want to include this in #169 since it's not really related, but I just noticed it when going through the code. The only place where FORCE and FORCEALL are really used is around line 509, but regardless of which option you have set the outcome is identical:

```bash
if [ "${FORCE}" != "yes" -a "${FORCEALL}" != "yes" ]; then
	sha1sum --status -c "${FILE_SHA}"
	if [ $? -eq 0 ]; then
		info "File already exists (${FILENAME}), won't download."
		if [ "${AUTOINSTALL}" != "yes" ]; then
			exit 0
		fi
		SKIP_DOWNLOAD="yes"
	else
		info "File exists but fails checksum. Redownloading."
		SKIP_DOWNLOAD="no"
	fi
elif [ "${FORCEALL}" == "yes" ]; then
	info "Note! File exists, but asked to overwrite with new copy"
else
	sha1sum --status -c "${FILE_SHA}"
	if [ $? -ne 0 ]; then
		info "File exists but fails checksum. Redownloading."
	else
		info "File exists and checksum passes, won't redownload."
		if [ "${AUTOINSTALL}" != "yes" ]; then
			exit 0
		fi
		SKIP_DOWNLOAD="yes"
	fi
fi
```

If you ignore the `elif`, the `if` and the `else` do the exact same thing, but with essentially reversed logic.

So am I missing something here or it safe to just pull out FORCEALL altogether (maybe leave it as an alias to -f for backwards compatibility)?

**Comment by mrworf** [comment]
I think at one point we only checked if the file exists in one of the cases. Either way, looking at the way things work now, we should deprecate the -F option, the -f is referenced as a means to force download when you have the same version. 

If you check 858a4293 you'll find that it wasn't using shasum on the first test, only the last :)

**Comment by demonbane** [comment]
Fixed in 8383048c013bf5f432fea818d147f84b81a4a3b0
